scripts/mkcloud

```diff
@@ -605,11 +602,36 @@ function restoreadminfromsnapshot()
     createadminsnapshot
 }
 
+function nodes()
+{
+    local query=${1:number}
+    local type=${2:all}
```

Comment:
No, gating passed :)
If you call the builtin you would write `type` and if you want the variable you write `$type`.
Btw. we also have other places where we use `$type` in mkcloud.